packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js

```diff
@@ -413,6 +416,15 @@ describe('plugin-meetings', () => {
         assert.strictEqual(mqeData.audioTransmit[0].common.transportType, 'TLS');
         assert.strictEqual(mqeData.videoReceive[0].common.transportType, 'TLS');
       });
+
```

Comment:
We will need a test to see what happens when the track label does not exist

Reply:
Added.